app/scripts/modules/titus/src/pipeline/stages/runJob/TitusRunJobStageConfig.tsx

```diff
@@ -0,0 +1,383 @@
+            value={stage.cluster.resources.disk}
+            required={true}
+          />
+        </StageConfigField>
+
+        <StageConfigField label="Network (Mbps)" helpKey="titus.deploy.network">
+          <input
+            type="number"
+            className="form-control input-sm"
+            onChange={e => this.stageFieldChanged('cluster.resources.networkMbps', e.target.value)}
+            value={stage.cluster.resources.networkMbps}
+            required={true}
+          />
+        </StageConfigField>
+
+        <StageConfigField label="GPU(s)" helpKey="titus.deploy.gpu">
+          <input
+            type="number"
+            className="form-control input-sm"
+            onChange={e => this.stageFieldChanged('cluster.resources.gpu', e.target.value)}
+            value={stage.cluster.resources.gpu}
+            required={true}
+          />
+        </StageConfigField>
+
+        <StageConfigField label="Entrypoint">
+          <input
+            type="text"
+            className="form-control input-sm"
+            value={stage.cluster.entryPoint}
+            onChange={e => this.stageFieldChanged('cluster.entryPoint', e.target.value)}
+          />
+        </StageConfigField>
+
+        <StageConfigField label="Runtime Limit (Seconds)" helpKey="titus.deploy.runtimeLimitSecs">
+          <input
+            type="number"
+            className="form-control input-sm"
+            value={stage.cluster.runtimeLimitSecs}
+            onChange={e => this.stageFieldChanged('cluster.runtimeLimitSecs', e.target.value)}
+            min="1"
+            required={true}
+          />
+        </StageConfigField>
+
+        <StageConfigField label="Retries" helpKey="titus.deploy.retries">
+          <input
+            type="number"
+            className="form-control input-sm"
+            onChange={e => this.stageFieldChanged('cluster.retries', e.target.value)}
+            value={stage.cluster.retries}
+            min="0"
+            required={true}
+          />
+        </StageConfigField>
+
+        <StageConfigField label="Property File" helpKey="titus.deploy.propertyFile">
+          <input
+            type="text"
+            className="form-control input-sm"
+            onChange={e => this.stageFieldChanged('propertyFile', e.target.value)}
+            value={stage.propertyFile}
+          />
+        </StageConfigField>
+
+        <div className="form-group">
+          <div className="col-md-9 col-md-offset-1">
+            <div className="checkbox">
+              <label>
+                <input
+                  type="checkbox"
+                  checked={stage.showAdvancedOptions}
+                  onChange={e => this.stageFieldChanged('showAdvancedOptions', e.target.checked)}
+                />
+                <strong>Show Advanced Options</strong>
+              </label>
+            </div>
+          </div>
+        </div>
+
+        <div
```

Comment:
className={`${stage.showAdvancedOptions === true ? 'collapse.in' : 'collapse'}`}

Reply:
👍